### ISSUE TITLE
Activate on scroll rather then on load

### ISSUE THREAD

**Issue by rruprai** [michalsnik/aos#506]
If I enter `AOS.init()` then this will cause the library to start on page. I have adjusted the call to only load on the scroll event. However, I noticed it is still being initialized on page load. 

The reason I would like to do this onscroll is because search engines bots don't use the scroll event and I would like all elements to be visible to them. For SEO purposes, anything that is hidden is given a lower weight even though it is still visible.

## Questions
1. Am I missing something here? 
2. How can I be sure the library is only initialized on scroll?

## This is:
- Question

## Specifications

  - AOS version: 2.3.4
  - OS: Windows 10
  - Browser: Chrome 76.0.3809.132

## Expected Behavior
Only initialized AOS when the scroll event occurs.

## Actual Behavior
AOS is initialized on page load.

## Steps to Reproduce the Problem
1. Add [aos.css using CDN](https://cdnjs.cloudflare.com/ajax/libs/aos/2.3.4/aos.css) to the `<head>` 
2. Add [aos.js](https://cdnjs.cloudflare.com/ajax/libs/aos/2.3.4/aos.js) to the end of the `<body>`
3. Using jQuery detect if scroll event occurs, if so then initialized AOS else don't. 

## Possible Solution
1. `AOS.init()` should only be initialized on scroll event. 
2. Add a setting to detect scroll event so it can be passed into the `.init()`

## Code used to load on scroll (assumes latest jquery version is used)

`$(function() {
    $(window).on('scroll', function() {
       AOS.init();
   });
});`

**Comment by molerat619** [comment]
@rruprai have you found a solution to this?

**Comment by rruprai** [comment]
@molerat619 

No, the sample code I posted doesn't work. Any help on this would be appreciated.

**Comment by hocomadvies** [comment]
This worked for me :) https://github.com/michalsnik/aos/issues/635

**Comment by prageeth** [comment]
I ended up going for this appraoch:
```
npm install is-web-crawler
```

And in the application, simply initialising AOS with the `disable` flag when the current user agent represents a "known" crawler.
```
import { isCrawlerUserAgent } from 'is-web-crawler';

AOS.init({
    disable: isCrawlerUserAgent(),
});
```

Disclaimer: I created the `is-web-crawler` library, but simply for the purpose of this use case.
